vizro-core/examples/assets/css/card.css

```diff
@@ -0,0 +1,4 @@
+div.left_side .card_container {
```

Comment:
Changes will be reverted here - just for demo purposes on how control group headers can be styled